ARM-wvd-templates/AddVirtualMachinesToHostPool/AddVirtualMachinesTemplate.json

```diff
@@ -409,6 +409,27 @@
         "description": "System data is used for internal purposes, such as support preview features."
       },
       "defaultValue": {}
+    },
+    "securityType": {
+      "type": "string",
+      "metadata": {
+        "description": "Specifies the SecurityType of the virtual machine. It is set as TrustedLaunch to enable UefiSettings. Default: UefiSettings will not be enabled unless this property is set as TrustedLaunch."

```

Comment:
I think it would be helpful to include a link to the docs for this here https://docs.microsoft.com/en-us/azure/templates/microsoft.compute/2021-07-01/virtualmachines?tabs=bicep#securityprofile so the customer has more context about this.